### PR TITLE
add 441. Arranging Coins

### PR DESCRIPTION
**Pull request by tkkawa**
Please check my code.

### CODE REVIEW

File: 441_arranging-coins.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# time = O(logn)
+# space = O(1)
+# done time = 10m
+
+
+class Solution:
+
+    def arrangeCoins(self, n: int) -> int:
+        low = 1
+        high = n
```
Comment (ryuji0123) [inline 441_arranging-coins.py:11]: Set high as invalid value when you use binary search. See this https://twitter.com/meguru_comp/status/697008509376835584

**Review by ryuji0123 — CHANGES_REQUESTED** [review]

**Comment by tkkawa** [comment]
I modified the code.

**Review by ryuji0123 — CHANGES_REQUESTED** [review]
use megru-style

**Comment by tkkawa** [comment]
I modified the code.
Please check my code.

File: 441_arranging-coins.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# space = O(1)
+# done time = 10m
+
+
+class Solution:
+
+    def arrangeCoins(self, n: int) -> int:
+        low = 0
+        high = n + 1
+        res = 0
```
Comment (ryuji0123) [inline 441_arranging-coins.py:12]: res is unnecessary

**Review by ryuji0123 — CHANGES_REQUESTED** [review]

**Comment by tkkawa** [comment]
I modified the code.
Please check my code.

**Review by kobakobashu — APPROVED** [review]
LGTM

**Pull request closed without merging**